### PR TITLE
[17.0][MIG] partner_category_security: Migration to version 17.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @tecnativa TT51234

@victoralmau @carlos-lopez-tecnativa please review

### CODE REVIEW

File: partner_category_security/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from lxml import etree
+
+from odoo import api, models
+
+# from odoo.addons.base.models.ir_ui_view import (
+#     transfer_modifiers_to_node,
+#     transfer_node_to_modifiers,
+# )
```
Comment (carlos-lopez-tecnativa) [inline partner_category_security/models/res_partner.py]: Please remove it if it is not used.

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM, tested functionality, just a minor change.

File: partner_category_security/models/res_partner.py (hunk 1 above)
Comment (pilarvargas-tecnativa) [reply partner_category_security/models/res_partner.py]: thanks!

**Review by victoralmau — APPROVED** [review]
Functional review OK

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**